stdlib/float64.gr

```diff
@@ -10,10 +14,41 @@ import {
   coerceFloat64ToNumber as toNumber
 } from "runtime/numbers"
 
+/**
+ * @section Values: Functions for working with the Float64 data type.
+ */
+
+/**
+ * Converts a Number to a Float64.
+ *
+ * @param number: The Number to convert
+ * @returns The Number represented as a Float64
+ * 
+ * @since v0.2.0
+ * @history v0.3.0: Function name changed from coerceNumberToFloat64 to fromNumber
+ */
 export fromNumber
-export toNumber
 
+/**
+ * Converts a Float64 to a Number.
+ *
+ * @param float: The Float64 to convert
+ * @returns The Float64 represented as a Number
+ * 
+ * @since v0.2.0
+ * @history v0.3.0: Function name changed from coerceFloat64ToNumber to toNumber
+ */
+export toNumber
 
```

Comment:
If we keep `Constants` and `Comparisons` sections, I'd want a section here for `Operations`

Reply:
@spotandjake please remember to add the `Operations` section above the `add` function here.

Reply:
Just committed it now.